docs/guides/livestreaming/webhook.mdx

```diff
@@ -1,23 +1,43 @@
 ---
-title: Using webhooks
+title: Webhooks
 description: Use webhooks to listen for video lifecycle events
 metaTitle: Using webhooks - Livepeer Studio
 metaDescription: Use webhooks to listen for video lifecycle events
 ---
 
-# **Using Webhooks**
+The following includes information and instructions on how to work with Webhooks to communicate with your application:
 
-Livepeer Studio uses webhooks to communicate with your application asynchronously
-when events for your stream occur. For example, you may want to know when a
-stream has become active or idle, so that you can surface this information to
-viewers. When these events happen, we’ll make a `POST` request to a URL that you
-specify.
+- Use webhooks to listen for video lifecycle events
+- Check webhook signatures using webhook signatures to ensure incoming webhook requests are coming from Livepeer
 
-### How to configure endpoints
 
-Webhook endpoints are specific for each `event`. Once the event is registered,
-all streams for this account will be triggering the specific event. Here is an
-example of registering for the `stream.started` and `stream.idle` events:
+## What you need to Know
+
+- Familiarity with [Livepeer Studio core concepts](/aboutstudio)
+- Experience working with `cURL`
+- Familiarity working with webhooks
+
+## Technical Preparation
+
+1. [A Livepeer account](https://livepeer.studio/login)
+1. [An API key](https://livepeer.studio/docs/guides/start-live-streaming/api-key) (Bearer)
+1. [cURL](https://curl.se/)
+1. Access to the [API Reference](https://livepeer.studio/docs/api-reference)
+
+
+## **About Webhooks**
+
+Livepeer Studio uses webhooks to communicate with your application asynchronously when events for your stream occur. For example, you may want to know when a stream has become active or idle, so that you can surface this information to viewers. When these events happen, you can make a `POST` request to a URL that you specify.
+
+## Configuring Webhook Endpoints
+
+Webhook endpoints are specific for each `event`. Once the event is registered, all streams for this account will be triggering the specific event. 
+
+**For Example:** Registering for `stream.started` and `stream.idle` events:
+
+### Request
+
+This cURL request registers a `stream.started` and `stream.idle` event (with a Bearer API key) so that all streams will trigger the `stream.started` and `stream.idle` events for this account.
```

Comment:
```suggestion
This cURL request registers a `stream.started` and `stream.idle` event so that all streams will trigger the `stream.started` and `stream.idle` events for this account.
```